packages/table/test/columnTests.tsx

```diff
@@ -51,4 +51,41 @@ describe("Column", () => {
         // TODO: re-enable once other PR merges
         // expect(table.find(".bp-table-column-name-text", 2).text()).to.equal("C");
     });
+
+    it("renders correctly with loading options", () => {
+        const cellValue = "my cell value";
+        const renderCell = () => <Cell>{cellValue}</Cell>;
+        const table = harness.mount(
+            <Table numRows={5}>
+                <Column name="Zero" loadingOptions={[ ColumnLoadingOption.CELLS ]} renderCell={renderCell} />
+                <Column
+                    name="One"
+                    loadingOptions={[ ColumnLoadingOption.CELLS, ColumnLoadingOption.HEADER ]}
+                    renderCell={renderCell}
+                />
+                <Column name="Two" renderCell={renderCell} />
+            </Table>,
+        );
+
+        const columnHeaderSelector = ".bp-table-column-headers .bp-table-header";
+        expect(table.find(columnHeaderSelector, 0).text()).to.equal("Zero");
+        expect(table.find(columnHeaderSelector, 1).text()).to.equal("");
+        expect(table.find(columnHeaderSelector, 2).text()).to.equal("Two");
+
+        const col0cells = table.element.querySelectorAll(".bp-table-cell-col-0");
+        for (let i = 0; i < col0cells.length; i++) {
+            expect(col0cells.item(i).textContent).to.equal("");
+        }
+
+        const col1cells = table.element.querySelectorAll(".bp-table-cell-col-1");
+        for (let i = 0; i < col1cells.length; i++) {
+            expect(col1cells.item(i).textContent).to.equal("");
```

Comment:
does it make more sense to test _instead_ that each of these cells contains a `.pt-skeleton`? seems more reliable and logical than asserting that empty text means they are in the loading state.